src/main/resources/openapi.json

```diff
@@ -1,286 +1,1279 @@
 {
-  "openapi": "3.0.2",
-  "info": {
-    "title": "HTTP Kafka bridge",
-    "version": "1.0.0"
-  },
-  "paths": {
-    "/topics/{topicname}": {
-      "post": {
-        "description": "Write to topic",
-        "operationId": "send",
-        "responses": {
-          "200": {
-            "description": "OK",
-            "content": {
-              "application/json": {}
-            }
-          }
-        }
-      },
-      "parameters": [
-        {
-          "name": "topicname",
-          "in": "path",
-          "description": "Topic name",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
-        }
-      ]
-    },
-    "/topics/{topicname}/partitions/{partitionid}": {
-      "post": {
-        "operationId": "sendToPartition",
-        "responses": {
-          "200": {
-            "description": "OK"
-          }
-        }
-      },
-      "parameters": [
-        {
-          "name": "topicname",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
+    "openapi": "3.0.0",
+    "info": {
+        "title": "Strimzi HTTP Bridge for Apache Kafka",
+        "description": "The Strimzi HTTP Bridge for Apache Kafka provides a REST API for integrating HTTP based client applications with a Kafka cluster. You can use the API to create and manage consumers and send and receive records over HTTP rather than the native Kafka protocol. The HTTP Bridge can be deployed on bare metal or in a container running on Kubernetes or OpenShift platform.",
+        "termsOfService": "http://swagger.io/terms/",
+        "contact": {
+            "email": "strimzi@redhat.com"
         },
-        {
-          "name": "partitionid",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "integer"
-          }
-        }
-      ]
+        "license": {
+            "name": "Apache 2.0",
+            "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+        },
+        "version": "1.0.0"
```

Comment:
Do we consider this API to be as version 1.0.0? We're defined _all_ the endpoints and we're sure they're all correct, even though this will be the initial release?

Reply:
Good point I would say "0.1.0"